tools/explorer/lib/ontology.py

```diff
@@ -17,6 +17,9 @@ class Ontology(object):
      Attributes:
           universe: A ConfigUniverse object detailing the various universes in
             the ontology
+          manager: An EntityTypeManager object to find greatest common subsets
+            of fields between entity types and complete lists of inheritied
+            fields for an entity type. This is primary used for _CreateMatch()
 
```

Comment:
is this important?

Reply:
The entity type manager is crucial for GetEntityTypesFromFields()